src/graph/graph.c

```diff
@@ -1322,36 +1428,38 @@ bool Graph_RelationshipContainsMultiEdge
 	ASSERT(Graph_RelationTypeCount(g) > r);
 	GrB_Index nvals;
 	// A relationship matrix contains multi-edge if nvals < number of edges with type r.
-	RG_Matrix R = Graph_GetRelationMatrix(g, r, transpose);
-	RG_Matrix_nvals(&nvals, R);
+	Delta_Matrix R = Graph_GetRelationMatrix(g, r, transpose);
+	Delta_Matrix_nvals(&nvals, R);
 
 	return (Graph_RelationEdgeCount(g, r) > nvals);
```

Comment:
I think it would be a bit cheaper to simply return `nvals > 0` where `nvals` is the number of entries within the multi-edge matrix, if it exists.